tests/servers/tests.py

```diff
@@ -27,6 +29,34 @@
 }
 
 
+class LiveServerThreadTest(TestCase):
+
+    def run_live_server_thread(self, connections_override=None):
+        thread = LiveServerTestCase._create_server_thread(connections_override)
+
+        thread.daemon = True
+        thread.start()
+        thread.is_ready.wait()
+        thread.terminate()
+        thread.join()
+
+    @unittest.skipIf(connection.vendor == 'sqlite', "sqlite does not permit closing connections")
```

Comment:
@charettes Yes, the "in-memory" property doesn't seem to be known at import time, so I used `SkipTest`. Patch updated.